contentcuration/contentcuration/static/js/edit_channel/tree_edit/hbtemplates/content_list_item.handlebars

```diff
@@ -1,84 +1,92 @@
-			<span class="metadata_to_hide row">
-				{{#if isfolder}}
-					{{#if node.metadata.coach_count}}
-						<span class="visibility pull-left" title="{{formatMessage (intlGet 'messages.coach_topic_title')}}">
-							<i class="icon-blue material-icons">local_library</i>
-							({{node.metadata.coach_count}})
+	</div>
+	<div class="col-xs-8 item_section main_item_section {{#if isfolder}}open_folder{{else}}open_file{{/if}}">
+		<h6 class="content_metadata_summary hide_on_drag {{#unless isfolder}}open_file{{/unless}} metadata_to_hide">
+			<!-- NEW/UPDATED/INCOMPLETE Indicators -->
+			<div class="pull-right metadata-indicator">
+				{{#if edit_mode}}
+					{{#if node.changed}}
+						<span class="unpublished" title="{{#if node.published}}{{formatMessage (intlGet 'messages.updated')}}{{else}}{{formatMessage (intlGet 'messages.created')}}{{/if}} {{formatRelative node.modified}}">
+							{{#if node.published}}{{formatMessage (intlGet 'messages.updated')}} {{else}}{{formatMessage (intlGet 'messages.new')}}{{/if}}
 						</span>
 					{{/if}}
+					{{#unless node.valid}}
+						<span class="invalid">{{formatMessage (intlGet 'messages.incomplete')}}</span>
+					{{/unless}}
 				{{/if}}
-				{{#if time}}&nbsp;<i class="material-icons pull-left">access_time</i>{{time}}{{/if}}
-				{{#if isfolder}}<span class="pull-left">&nbsp;{{formatMessage (intlGet (concat 'messages.resource_count')) count=count}}</span>{{/if}}
-				{{#unless isfolder}}
-					<span class="pull-left">
-						&nbsp;
-						{{#if isexercise}}
-							<i class="material-icons">check_circle</i>
-							{{formatMessage (intlGet (concat 'messages.question_count')) count=count}}
-						{{else}}
-							<i class="material-icons">insert_drive_file</i>
-							{{{format_file_size node.metadata.resource_size}}}
-						{{/if}}
-					</span>
-				{{/unless}}
-				{{#if node.changed}}
-					<span class="pull-left">
-						&nbsp;
-						{{#if node.published}}<em class="unpublished" title="{{formatMessage (intlGet 'messages.updated')}} {{formatRelative node.modified}}">{{formatMessage (intlGet 'messages.updated')}}</em>
-						{{else}}<em class="unpublished" title="{{formatMessage (intlGet 'messages.created')}} {{formatRelative node.modified}}">{{formatMessage (intlGet 'messages.new')}}</em>
-						{{/if}}
-					</span>
+			</div>
+
+
+			<!-- Content Kind -->
+			{{#unless isfolder}}
+			<div class="metadata-indicator">
+				<span class="content-kind kind-{{node.kind}}"><i class="material-icons">{{{get_icon node.kind}}}</i> {{{translate node.kind}}}</span>
+			</div>
+			{{/unless}}
+
+			<!-- Resource count -->
+			{{#if isfolder}}
+				<div class="metadata-indicator resource_count">
+					{{formatMessage (intlGet (concat 'messages.resource_count')) count=count}}
+				</div>
+			{{/if}}
+
+			<!-- Coach Content -->
+			{{#if isfolder}}
+				{{#if node.metadata.coach_count}}
+					<b class="visibility" title="{{formatMessage (intlGet 'messages.coach_topic_title')}}">
```

Comment:
similarly, `b` has been deprecated in favor of `strong`, although I'd argue that "bold" is semantically more expressive than "italicized"